### ISSUE TITLE
error in RMSNorm documentation

### ISSUE THREAD

**Issue by yuanyao-nv** [pytorch/pytorch#144377]
### 📚 The doc issue

The formula for RMS [documentation ](https://pytorch.org/docs/stable/generated/torch.nn.modules.normalization.RMSNorm.html) should have MS instead of RMS on the denominator. Writing RMS inside sqrt implies there are two sqrt operations.
![image](https://github.com/user-attachments/assets/631e1675-9503-4e06-8afa-688305adb9f2)

**Comment by awgu** [comment]
This is fixed a while ago in `main` branch and will be in 2.6.

https://pytorch.org/docs/2.6/generated/torch.nn.modules.normalization.RMSNorm.html
